pylint/lint/run.py

```diff
@@ -227,6 +227,7 @@ def __init__(
             elif linter.config.jobs == 0:
                 linter.config.jobs = _cpu_count()
 
+        linter.verbose = self.verbose
```

Comment:
What do you think about doing this in `_config_initialization`? Or maybe set it on the `config` object in that function?

Reply:
I had to hunt around to find where it belonged, and this my best first guess.  :-)  Happy to put it wherever.

Reply:
@Pierre-Sassoulas opinion?

Reply:
Intuitively I would move this to line 177 so it's verbose sooner. Now that I read the code it would make sense to do it in ``_config_initialization`` but there's a ton of things we do directly in the ``Run`` class which is maybe why it's not intuitive ?

Reply:
Moved it to `_config_initialization` -- yep, hard to find the right spot.  Line 177 would have been fine b/c we just pass the boolean and then use it in the initialization function, but doesn't really matter.

Reply:
moving it to config_init causes `pylint/config/config_initialization.py:26:0: R0915: Too many statements (51/50) (too-many-statements)`

Reply:
Need guidance here, @Pierre-Sassoulas and @DanielNoord.

Reply:
Ah my bad that’s already fixed, just saw the commit. Thx 👍